docs/config.md

```diff
@@ -28,6 +28,7 @@ A `static.config.js` file is optional, but recommended at your project root to u
 - [disablePreload](#disablePreload)
 - [babelExcludes](#babelExcludes)
 - [productionSourceMaps](#productionSourceMaps)
+- [productionLog](#productionLog)
```

Comment:
Should probably change to silent

Reply:
Ooops... My bad. Fixed in the last commit.